charts/stable/jellyfin/values.yaml

```diff
@@ -32,4 +44,53 @@ workload:
       containers:
         main:
           env:
-            JELLYFIN_PublishedServerUrl: "https://jelly.mydomain.com"
+            JELLYFIN_PublishedServerUrl: "{{ $.Values.chartContext.APPURL }}"
+  broadcastproxy:
+    enabled: false
```

Comment:
I would like this to be enabled in CI, so we catch early major issues.

Reply:
Would you like me to submit another PR with proxy enabled by default?